suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -71,7 +71,7 @@ export const TransactionListItemValues = ({
                         <SignValueFormatter value={getTransactionValueSign(transaction.type)} />
                     )}
                     <CryptoToFiatAmountFormatter
-                        value={transaction.amount}
+                        value={Math.abs(Number(transaction.amount))}
```

Comment:
Why is the `Math.abs` needed here and also in `TokenToFiatAmountFormatter` now? Is it because of some specific feature of Solana?

What about modyfing the `...Formatter` components itself, so the `.abs()` is applied there? It should prevent us from needing the `abs()` anywhere else in the future.

Reply:
for solana `-` sign is part of the amount which causes visual inconsistencies (double -). This is similar mitigation that desktop does.

Reply:
🤔  but it would be needed on more places and it would be better to handle separately. I'll remove this and create issue

Reply:
https://github.com/trezor/trezor-suite/issues/15424